pmg/user_management.py

```diff
@@ -111,11 +112,12 @@ def email_alerts():
 
         # register a google analytics event
         ga_event('user', 'change-alerts')
-        flash("Your notification settings have been updated successfully.", "success")
-        if next_url:
-            return redirect(next_url)
 
-        return redirect(url_for('email_alerts'))
+        # When was this called?
+        # if next_url:
```

Comment:
I think so, yes. When a user clicks the 'email alert' button on a committee page, it goes through this path. It would be nicer to do it via ajax without a page submit, but that's okay for now.